src/components/DownloadLoginFile.js

```diff
@@ -166,8 +184,16 @@ class DownloadLoginFile extends Nimiq.Observable {
 
             // If window gets blurred, show 'Continue' button in interface and do not automatically
             // consider the download successful.
-            if (!document.hasFocus()) {
+            // As mobile Safari on iOS 13.x+ does support the download attribute, it no longer uses the long tap
+            // fallback. However, if the page changes its hash in the background while the prompt asking to download
+            // the file was not confirmed yet, the download will simply do nothing. To prevent that behaviour the
+            // hash change is delayed by the .maybe-downloaded confirmation mechanism.
+            if (!document.hasFocus()
+                || (BrowserDetection.isIOS()
+                    && BrowserDetection.isSafari()
+                    && BrowserDetection.iOSVersion()[0] > 12)) {
```

Comment:
I think we should test whether we need to include iOS Chrome too.
Or didn't we even say we want to have the `Continue` button always on mobile?

Reply:
Actually I think we should probably just get rid of the `BrowserDetection.isSafari()` condition.
For non iOS devices I think the continue button is simply not necessary, but on iOS 13 we can just declare this as default behaviour, no matter what browser.